### DISCUSSION TITLE
Just a potential problem about our issues template

### DISCUSSION THREAD

**Discussion by Superskyyy** [apache/skywalking#8103] [General]
Now our issues template offers three checkboxes to record some initial inputs on agreements and willingness to submit a PR. 
![image](https://user-images.githubusercontent.com/26076517/141056475-536b6bf8-a7ec-44ef-8594-0bab7f3b2939.png)

It is great until GitHub decides to visualize them as "progress" in the issues list. Referring to the screenshot below, such checkboxes are making the issue **appear to be worked on** - `2 of 3 tasks` done, while often misleading in our use case (imagine a newcomer filtering for good first issues and seeing there's already progress on certain tasks). Shall we change it to a dropdown list or something else? 
![image](https://user-images.githubusercontent.com/26076517/141055927-ebcc83bc-54f4-43c3-93a9-a261cf509453.png)

There's ongoing feedback for this problem, we probably could get a workaround now, and switch back when it gets fixed by GitHub in the future. https://github.com/github/feedback/discussions/5238

**Comment by wu-sheng** [comment]
This is a mislead if you read it. I am just not sure whether they will.😄
After all, the content is clear.

Anyway, changing it to a selector seems easy, if you feel that better, do it.
